.github/workflows/gpu.yml

```diff
@@ -20,20 +19,33 @@ jobs:
         build_type: [Release]
         compiler: [{c: gcc, cxx: g++}]
         shared_library: ['ON', 'OFF']
-    # TODO enable testing on DSS-WINDOWS
-    runs-on:  [ "DSS-LEVEL_ZERO", "DSS-UBUNTU" ]
+        include:
+          - os-labels: ["DSS-LEVEL_ZERO", "DSS-UBUNTU"]
+            os: 'UBUNTU'
+            compiler: [{c: gcc, cxx: g++}]
+            cmake_toolchain_file: 'OFF'
+            level_zero_provider: 'OFF'
```

Comment:
heh, I hope this is only for debug - this should not be OFF 😉

Reply:
Done